packages/quantic/force-app/main/default/lwc/quanticSort/quanticSort.js

```diff
@@ -75,11 +81,22 @@ export default class QuanticSort extends LightningElement {
   schemas = {};
 
   labels = {
-    sortBy,
     relevancy,
     newest,
     oldest,
     invalidCustomSortConfig,
+    sortBy,
+  };
+
+  variants = {
+    default: {
+      name: 'default',
+      label: sortBy,
+    },
+    wide: {
+      name: 'wide',
+      label: undefined,
```

Comment:
I would rather have an empty string than an explicit `undefined`.

the reason is, if you call your `get label()` and put it in a template like so: `<p>{label}</p>` it will write `undefined` in the page like so `<p>'undefined'</p>` instead of an empty string.

```suggestion
      label: '',
```

Reply:
I can change it to "" if you want, but leaving undefined will not render `undefined` on the page. I tested it here:
it simply renders nothing akin to an empty string

https://github.com/user-attachments/assets/d0c3b9ae-5b21-4516-bc7a-cd2f10e5a169